.gitignore

```diff
@@ -5,6 +5,14 @@
 
 #ignore backups
 *~
+cmake/**
```

Comment:
that sounds like a terrible idea?

Reply:
wups, I did that in because I always got tons of unneccessary files from this folder to potentially commit and had to singlepick every file. I'll remove that